features/wishlists.feature

```diff
@@ -157,4 +157,19 @@ Scenario: Edit a product in a wishlist
     When I press the "search-wishlist" button
     Then I should see "New Board game" in the results
     And I should see "6666" in the results
-    And I should not see "30" in the results
\ No newline at end of file
+    And I should not see "30" in the results
+
+Scenario: Place a product from a wishlist in a shopcart
+    When I visit the "Home Page"
+    And I press the "order-product" button
+    Then I should see the message "Product id and wishlist id should be defined"
+    When I press the "search-wishlist" button
+    Then I should see "Pen" in the results
+    When I copy the product id of "Pen"
+    And I paste the "wishlist-product-id" field
+    And I copy the wishlist id of "Pen"
+    And I paste the "wishlist-id" field
+    And I press the "order-product" button
+    Then I should see the message "Success"
+    When I press the "search-wishlist" button
+    Then I should see "IN_CART" in the results
```

Comment:
This makes no sense.
If you ```search-wishlist```, "IN_CART" will  be always in the results because every product has an attribute called ```in_cart_status```.
So I think it is better to search that ordered product and check the ordered status.